### PR TITLE
fix: add missing posting to daily extensions

### PR DESCRIPTION
**Pull request by karolsojko**

### CODE REVIEW

File: src/Controller/ItemsController.ts [hunk 1]
```diff
@@ -28,11 +34,25 @@ export class ItemsController extends BaseHttpController {
...
-      userUuid: response.locals.user.uuid,
-      itemHashes: request.body.items,
-    })
+    try {
+      await this.postToRealtimeExtensions.execute({
+        userUuid: response.locals.user.uuid,
+        itemHashes: request.body.items,
+      })
+
+      await this.postToDailyExtensions.execute({
```
Comment (atmoio) [inline src/Controller/ItemsController.ts:43]: Hmm I don't think we should posting to daily extensions during sync?

File: src/Controller/ItemsController.ts [hunk 2]
```diff
@@ -28,11 +34,25 @@ export class ItemsController extends BaseHttpController {
       apiVersion: request.body.api,
     })
 
-    await this.postToRealtimeExtensions.execute({
-      userUuid: response.locals.user.uuid,
-      itemHashes: request.body.items,
-    })
+    try {
+      await this.postToRealtimeExtensions.execute({
```
Comment (atmoio) [inline src/Controller/ItemsController.ts:38]: Will this await the full execution of the job, or just queuing the job?

Comment (atmoio) [reply src/Controller/ItemsController.ts:38]: If awaiting full execution that would be unideal as communicating with 3rd party clouds could take arbitrarily long.

File: src/Controller/ItemsController.ts (hunk 1 above)
Comment (karolsojko) [reply src/Controller/ItemsController.ts:43]: We should, vide: https://github.com/standardnotes/syncing-server/blob/develop/app/controllers/api/items_controller.rb#L17-L20

File: src/Controller/ItemsController.ts (hunk 2 above)
Comment (karolsojko) [reply src/Controller/ItemsController.ts:38]: Adding event to the queue. In essence this replicates: https://github.com/standardnotes/syncing-server/blob/develop/app/controllers/api/items_controller.rb#L17-L20

Comment (karolsojko) [reply src/Controller/ItemsController.ts:38]: > If awaiting full execution that would be unideal as communicating with 3rd party clouds could take arbitrarily long.

I would know ;)

Comment (atmoio) [reply src/Controller/ItemsController.ts:38]: Ah cool 👌

**Pull request merged**